src/components/Icons/Icon.stories.tsx

```diff
@@ -11,13 +11,17 @@ export default {
 };
 
 const iconRow = function (array, icon, modifiers = "") {
+    let mods = [modifiers];
+    if (icon.indexOf("_negative") !== -1) {
```

Comment:
if the black background should be in Design System (which I am not sure it should be?  Is it possible that we want the white icon on a background that isn't black?)  I think we should ask the user to pass a "black-background" or "negative-icon" modifier in directly rather than doing this off of the name of the icon.

Reply:
We definitely don't want the black background on the white icon, so I'm going to resolve this

Reply:
The last update should resolve it, but now the example color for the background is still black. This is just not part of the modifiers anymore.

Reply:
I think that's great 👍🏻